src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/script.js.template

```diff
@@ -640,6 +651,181 @@ document.addEventListener("DOMContentLoaded", function(e) {
         }
         window.addEventListener("scroll", handleScroll);
     }
+
+    function allTocNavs() {
+        return Array.from(document.querySelectorAll("nav.toc"));
+    }
+
+    function nestedTocLists(scope) {
+        const listsToSort = [];
+        const sectionLinks = scope.querySelectorAll("ol.toc-list > li > a");
+
+        sectionLinks.forEach(function(link) {
+            const href = link.getAttribute("href");
+            if (href === "#constructor-detail" || href === "#method-detail"
+                || href === "#field-detail") {
```

Comment:
I think this is missing a few kinds of detail sections: `#annotation-interface-element-detail`, `#enum-constant-detail`, `#property-detail`.